core/src/test/scala/org/http4s/rho/ParamDefaultValueSpec.scala

```diff
@@ -6,340 +6,340 @@ import org.specs2.mutable.Specification
 
 class ParamDefaultValueSpec extends Specification {
 
-  def body(service: HttpRoutes[IO], r: Request[IO]): String =
-    new String(service(r).value.unsafeRunSync().getOrElse(Response.notFound).body.compile.toVector.unsafeRunSync().foldLeft(Array[Byte]())(_ :+ _))
+  def body(routes: HttpRoutes[IO], r: Request[IO]): String =
+    new String(routes(r).value.unsafeRunSync().getOrElse(Response.notFound).body.compile.toVector.unsafeRunSync().foldLeft(Array[Byte]())(_ :+ _))
 
   def requestGet(s: String, h: Header*): Request[IO] =
     Request(bits.MethodAliases.GET, Uri.fromString(s).right.getOrElse(sys.error("Failed.")), headers = Headers(h: _*))
 
   "GET /test1" should {
-    val service = new RhoService[IO] {
+    val routes = new RhoService[IO] {
```

Comment:
This is still a `RhoService`, so i don't know that the rename is appropriate here.